pkg/script/script_test.go

```diff
@@ -44,6 +44,19 @@ func TestBuildBootstrapScriptSad(t *testing.T) {
 	}
 }
 
+func TestBuildBootstrapScriptDoesNotExceed16KB(t *testing.T) {
+	scripts := []string{
+		"resources/amazon_k8s_ubuntu_16.04_master.sh",
```

Comment:
Would it make sense to generate a random file on some temporary location that is bigger than 16kb rather than committing such file? Would it be even possible? I believe that could help to avoid such confusions.